frontend/ic_vetkeys/src/utils/utils.ts

```diff
@@ -306,6 +306,34 @@ export function augmentedHashToG1(
     return pt;
 }
 
+/**
+ * Verify a BLS signature
+ *
+ * A VetKey is in the end a valid BLS signature; this function checks that a
+ * provided BLS signature is the valid one for the provided public key and
+ * message.
+ *
+ * When a VetKey struct is created (VetKey.decryptAndVerify) the signature (aka
+ * vetkey) is already verified, so using this function is only necessary when
```

Comment:
maybe add ", which can be obtained via `VetKey::signatureBytes`, "

nit: maybe also write "vetKey" instead of "vetkey" whenever we are not talking about the TypeScript class VetKey (or Rust struct VetKey).

Reply:
Fixed